new/LIT6620DerHomKah.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6620DerHomKah" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘ፳ወ፬፡ ካህናተ፡ ሰማይ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Dǝrsān za20wa4 kāhǝnāta samāy</title>
+                <title xml:lang="en" xml:id="t2">Homily for the commemoration of the Twenty-Four Elders of Heaven</title>
+                <editor role="generalEditor" key="AB"/>
```

Comment:
maybe add as a title variant "... heavenly priests" as they are often referred to that way